expression/aggregation/base_func_test.go

```diff
@@ -39,3 +39,14 @@ func (s *testBaseFuncSuite) TestClone(c *check.C) {
 	c.Assert(desc.Args[0], check.Equals, col)
 	c.Assert(desc.equal(s.ctx, cloned), check.IsFalse)
 }
+
+func (s *testBaseFuncSuite) TestMaxMin(c *check.C) {
+	col := &expression.Column{
+		UniqueID: 0,
+		RetType:  types.NewFieldType(mysql.TypeLonglong),
+	}
+	col.RetType.Flag |= mysql.NotNullFlag
+	desc, err := newBaseFuncDesc(s.ctx, ast.AggFuncMax, []expression.Expression{col})
+	c.Assert(err, check.IsNil)
+	c.Assert(desc.RetTp.Flag == mysql.NotNullFlag, check.IsFalse)
```

Comment:
We'd better check `mysql.HasNotNullFlag(desc.RetTp.Flag)`

Reply:
done